### DISCUSSION TITLE
Error loading project with video from Vimeo

### DISCUSSION THREAD

**Discussion by brunomartelli** [mozilla/hubs#2849] [General Questions & Discussion] [answered]
getting this error on 2 vimeo hosted videos, 2 other videos in same project are fine 
![image](https://user-images.githubusercontent.com/4471896/90070900-8d587a80-dcec-11ea-820d-4397f8cfa623.png)

**Comment by brunomartelli** [comment]
The only thing similarity between the videos is  - videos that are from screen recordings or from non standard resolution cameras will play. Videos shot on a DSLR - i.e. like real actual videos won't play. e.g 1920x1080  or 4k videos. The screen recording from the computer recorded at whatever random res it was,  then uploaded to vimeo are all fine. Sadly this is the opposite of what I need.

**Comment by brunomartelli** [comment]
Todays error message:


o: Error loading video https://vimeo.com/385288553:
  Cause:
    Error loading video "https://hubs-proxy.com/https://35vod-adaptive.akamaized.net/…5358f75b,66ddb939,f5492576,328844e7/master.mpd?base64_init=1":
      Cause:
        Failed to init decoder
    r errors.js:29
    o errors.js:37
    load VideoNode.js:133
    r errors.js:29
    o errors.js:37
    s Video.js:70
    r helpers.js:59

**Comment by brunomartelli** [comment]
Turns out if Vimeo video has sound it wont load

**Reply by davegoopot** [reply]
@brunomartelli  how do you mean a video with sound? I've added quite a few videos from Vimeo to Hubs without issue and they all had audio.

**Reply by BonzoMartini** [reply]
in Spoke 
![image](https://user-images.githubusercontent.com/66266175/92032784-91fbd600-ed62-11ea-9219-0d9718ddf035.png)
vimeo link https://vimeo.com/222647996

in Hubs:
![image](https://user-images.githubusercontent.com/66266175/92032945-ca031900-ed62-11ea-87b8-a38c1e70b5ae.png)

**Reply by BonzoMartini** [reply]
i have had no luck with any videos from Vimeo that have sound. They all don't load in Spoke only Hubs.

**Comment by emclaren** [comment]
Linking this related discussion - https://github.com/mozilla/hubs/discussions/2859

**Comment by emclaren** [comment, marked as answer]
This appears to be related to this outstanding Spoke bug mozilla/Spoke#964

**Reply by BonzoMartini** [reply]
yes
